dev/tsconfig.dev.json

```diff
@@ -9,11 +9,10 @@
     "paths": {
       // Alias to paths required for Sanity CLI to run in dev-mode
       "@sanity/mutator": ["./packages/@sanity/mutator/src"],
-      "@sanity/util/fs": ["./packages/@sanity/util/exports/fs.ts"],
-      "sanity/_internal": ["./packages/sanity/exports/_internal.ts"],
-      "sanity/migrate": ["./packages/sanity/exports/migrate.ts"],
-      "sanity/migrate/mutations": ["./packages/sanity/exports/migrate-mutations.ts"],
-      "sanity/cli": ["./packages/sanity/exports/cli.ts"]
+      "@sanity/util/fs": ["./packages/@sanity/util/src/_exports/fs.ts"],
```

Comment:
Yes, good catch! I knew I forgot something 🤔

Reply:
Added! In order for it to work I had to remove the `strict: false` used in `@sanity/schema` and handle implicit any's.
The `pnpm check:types` command now ensures that `paths` are setup correctly so that the IDE shows type changes without first having to run `pnpm build`.
It works across all packages.

Reply:
Amazing! Now that I think of it, I'm pretty sure the reason we used project references instead of paths in the first place was to be able to let packages have different `compilerOptions`. Great to have unified tsconfig!

Reply:
Ah ok! I'll see if I can add it back to the `check:types` commands!